lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -493,26 +371,26 @@ export class UserAgentApplication {
 
     // If already in progress, do not proceed
     // TODO: Should we throw or return an error here?
-    if (this._acquireTokenInProgress) {
+    if (this.acquireTokenInProgress) {
       return;
     }
 
     // If no session exists, prompt the user to login.
     if (!userObject && !(extraQueryParameters && (extraQueryParameters.indexOf(Constants.login_hint) !== -1 ))) {
     // TODO: This should be replaced with error callback
     // TODO: Is this always accessToken?
-      if (this._tokenReceivedCallback) {
-        this._logger.info("User login is required");
-        this._tokenReceivedCallback(ErrorDescription.userLoginError, null, ErrorCodes.userLoginError, Constants.accessToken, this.getUserState(this._cacheStorage.getItem(Constants.stateLogin, this.storeAuthStateInCookie)));
+      if (this.tokenReceivedCallback) {
+        this.logger.info("User login is required");
```

Comment:
yeah we call logger here, but above call this.config.system.logger

Reply:
The above is an oversight. This reference is what is correct. Will update this in the next checkin.